src/main/java/nitin/collections/d_maps/TreeMap/TreeMapComparatorPrimitiveKey.java

```diff
@@ -26,6 +26,4 @@ public int compare(Integer o1, Integer o2) {
             System.out.println("Key is : " + entry.getKey() + " And value is : " + entry.getValue());
         }
     }
-
-}
```

Comment:
Please remove empty lines

Reply:
invalid comment. The lines are already removed. Please check the LHS of the diff for latest changes

Reply:
revoew completed